srcdeps-core/src/main/java/org/l2x6/srcdeps/core/config/BuilderIo.java

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright 2015-2016 Maven Source Dependencies
+ * Plugin contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.l2x6.srcdeps.core.config;
+
+import java.lang.ProcessBuilder.Redirect;
+
+import org.l2x6.srcdeps.core.shell.IoRedirects;
+
+/**
+ * A triple of definitions how to handle the three standard I/O steams of a process.
+ *
+ * @see IoRedirects
+ * @author <a href="https://github.com/ppalaga">Peter Palaga</a>
+ */
+public class BuilderIo {
+
+    public static class Builder {
+
+        private String stderr = BuilderIoScheme.inherit.name();
+
+        private String stdin = BuilderIoScheme.inherit.name();
+
+        private String stdout = BuilderIoScheme.inherit.name();
+
+        public BuilderIo build() {
+            return new BuilderIo(stdin, stdout, stderr);
+        }
+
+        public Builder stderr(String stderr) {
+            this.stderr = stderr;
+            return this;
+        }
+
+        public Builder stdin(String stdin) {
+            this.stdin = stdin;
+            return this;
+        }
+
+        public Builder stdout(String stdout) {
+            this.stdout = stdout;
+            return this;
+        }
+    }
+
+    /**
+     * An enum of prefixes to use when encoding {@link Redirect}s as URI {@link String}s. See
+     * {@link IoRedirects#parseUri(String)}.
+     *
+     */
+    enum BuilderIoScheme {
+        append, err2out, inherit, read, write
```

Comment:
The delimiter is `:` hence the `write:/dev/null` should work. 

I admit I have not thought deeply about all the consequences and possibilities of the present IO configuration approach. I just tried to stringify `java.lang.ProcessBuilder.Redirect` because `ProcessBuilder.Redirect` is what I need to create the builder process. I do not think I tested once with something else than "redirect all". Integration tests in general are actually non-existent ATM too.